epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -93,12 +94,15 @@ public SourceVersion getSupportedSourceVersion() {
   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
     LinkedHashMap<TypeElement, ClassToGenerateInfo> modelClassMap = new LinkedHashMap<>();
 
-    for (Element attribute : roundEnv.getElementsAnnotatedWith(EpoxyAttribute.class)) {
-      try {
+    try {
+      for (Element attribute : roundEnv.getElementsAnnotatedWith(EpoxyAttribute.class)) {
         processAttribute(attribute, modelClassMap);
-      } catch (EpoxyProcessorException e) {
-        writeError(e);
       }
+      for (Element clazz : roundEnv.getElementsAnnotatedWith(EpoxyClass.class)) {
+        processClass((TypeElement) clazz, modelClassMap);
```

Comment:
I'd be fine with calling `getOrCreateTargetClass(modelClassMap, classElement)` directly here and getting rid of the `processClass` method, since that method is only called once and is just the one line. The naming adds some readability, but it also adds an extra step to follow